android-auto-app/build.gradle.kts

```diff
@@ -0,0 +1,61 @@
+plugins {
+  id("com.android.application")
+  kotlin("android")
+  kotlin("android.extensions")
+  id("com.mapbox.maps.token")
+}
+
+android {
+  compileSdkVersion(AndroidVersions.compileSdkVersion)
+  defaultConfig {
+    applicationId = "com.mapbox.maps.testapp.auto"
+    minSdkVersion(AndroidVersions.minAndroidAutoSdkVersion)
+    targetSdkVersion(AndroidVersions.targetSdkVersion)
+    versionCode = 1
+    versionName = "0.1.0"
+    multiDexEnabled = true
+    testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    testInstrumentationRunnerArguments = mapOf(
+      "clearPackageData" to "true"
+    )
+  }
+
+  compileOptions {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+  }
+
+  testOptions {
+    if (!project.hasProperty("android.injected.invoked.from.ide")) {
+      execution = "ANDROIDX_TEST_ORCHESTRATOR"
+    }
+  }
+}
+
+androidExtensions {
+  isExperimental = true
+}
+
+dependencies {
+  implementation(project(":extension-androidauto"))
+  implementation(project(":sdk"))
+  implementation(Dependencies.googleCarAppLibrary)
+  implementation(Dependencies.kotlin)
+  implementation(Dependencies.androidxAppCompat)
+  androidTestUtil(Dependencies.androidxOrchestrator)
+  androidTestImplementation(Dependencies.androidxTestRunner)
+  androidTestImplementation(Dependencies.androidxJUnitTestRules)
+  androidTestImplementation(Dependencies.androidxRules)
+  androidTestImplementation(Dependencies.androidxTestJUnit)
+  androidTestImplementation(Dependencies.androidxEspresso)
+  androidTestImplementation(Dependencies.androidxUiAutomator)
+  testImplementation(Dependencies.junit)
+}
+
+project.apply {
+  from("$rootDir/gradle/ktlint.gradle")
+  from("$rootDir/gradle/lint.gradle")
+}
+
+the<com.mapbox.AccessTokenExtension>().file =
+  "${project.rootDir}/extension-style-app/src/main/res/values/developer-config.xml"
```

Comment:
Pretty strange solution imho 😄

Reply:
This is generated 🤦  will remove.

Reply:
well, can not remove, this is the intended way to configure our access token plugin..

but good catch that the path is not correct, fixed it to `"${project.rootDir}/android-auto-app/src/main/res/values/developer-config.xml"`